src/js/core/methods/CardanoSignTransaction.js

```diff
@@ -46,6 +49,11 @@ const CardanoSignTransactionFeatures = Object.freeze({
     AuxiliaryDataHash: ['0', '2.4.2'],
     TokenMinting: ['0', '2.4.3'],
     Multisig: ['0', '2.4.3'],
+    NetworkIdInTxBody: ['0', '2.4.4'], // TODO adjust version number
+    OutputDatumHash: ['0', '2.4.4'], // TODO adjust version number
+    ScriptDataHash: ['0', '2.4.4'], // TODO adjust version number
+    Plutus: ['0', '2.4.4'], // TODO adjust version number
+    KeyHashStakeCredential: ['0', '2.4.4'], // TODO adjust version number
```

Comment:
Are these features ok, or do we want to merge some of them?

Reply:
I think it's better to have them separately.